src/components/appointments/AptModal.js

```diff
@@ -165,15 +165,17 @@ class AptModal extends Component {
   render () {
     if (!this.props.got_data) {
       return (
-        <Modal history={this.props.history} title=''>
+        <Modal history={this.props.history} title='' parent_el={this.props.config.modal_parent}>
```

Comment:
```suggestion
        <Modal history={this.props.history} title='' config={this.props.config}>
```
just pass config would be easier